src/cfi.rs

```diff
@@ -104,7 +103,7 @@ impl<'input, Endian> FallibleIterator for CfiEntriesIter<'input, Endian>
     type Item = CieOrFde<'input, Endian>;
     type Error = Error;
 
-    fn next(&mut self) -> Result<Option<Self::Item>, Self::Error> {
+    fn next(&mut self) -> ::std::result::Result<Option<Self::Item>, Self::Error> {
```

Comment:
The original motivation was that I'd kind of like this to fail loudly if we change any of the `next` signatures, but I suppose the type checker will do that either way. 

/me shrugs